test/functional/cmdLineTests/modularityddrtests/modularityddrtests.xml

```diff
@@ -372,5 +372,262 @@
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP' $DUMPFILE$" platforms="zos_390-31.*" />
+		<exec command="tso delete J9CORE.DMP.*" platforms="zos_390-64.*" />
+		<exec command="tso delete J9CORE.DMP" platforms="zos_390-31.*" />
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core</arg>
+			<arg>$DUMPFILE$</arg>
+			<input>!findmodules name thisdoesnotexist</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success">Found 0 modules</output>
+		<output regex="no" type="failure">!j9module</output>
+		<output regex="no" type="failure">DDRInteractiveCommandException</output>
+	</test>
+
+	<test id="Run !findmodules requires">
+		<exec command="sh" capture="LOGNAME" platforms="zos.*" >
+			<arg>-c</arg>
+			<arg>echo $$LOGNAME</arg>
+		</exec>
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP.X001' $DUMPFILE$" platforms="zos_390-64.*" />
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP' $DUMPFILE$" platforms="zos_390-31.*" />
+		<exec command="tso delete J9CORE.DMP.*" platforms="zos_390-64.*" />
+		<exec command="tso delete J9CORE.DMP" platforms="zos_390-31.*" />
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core</arg>
+			<arg>$DUMPFILE$</arg>
+			<input>!findmodules requires java.base</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success">jdk.compiler</output>
+		<output regex="no" type="required">!j9module 0x</output>
+		<output regex="no" type="failure">DDRInteractiveCommandException</output>
+	 </test>
+
+	 <test id="Run !findmodules requires on non-existing module">
+		<exec command="sh" capture="LOGNAME" platforms="zos.*" >
+			<arg>-c</arg>
+			<arg>echo $$LOGNAME</arg>
+		</exec>
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP.X001' $DUMPFILE$" platforms="zos_390-64.*" />
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP' $DUMPFILE$" platforms="zos_390-31.*" />
+		<exec command="tso delete J9CORE.DMP.*" platforms="zos_390-64.*" />
+		<exec command="tso delete J9CORE.DMP" platforms="zos_390-31.*" />
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core</arg>
+			<arg>$DUMPFILE$</arg>
+			<input>!findmodules requires thisdoesnotexist</input>
+			<input>quit</input>
+		</command>
+		<output regex="no" type="success">Found 0 modules</output>
+		<output regex="no" type="failure">!j9module</output>
+		<output regex="no" type="failure">DDRInteractiveCommandException</output>
+	</test>
+
+	<test id="Run !findmodules package">
+		<exec command="sh" capture="LOGNAME" platforms="zos.*" >
+			<arg>-c</arg>
+			<arg>echo $$LOGNAME</arg>
+		</exec>
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP.X001' $DUMPFILE$" platforms="zos_390-64.*" />
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP' $DUMPFILE$" platforms="zos_390-31.*" />
+		<exec command="tso delete J9CORE.DMP.*" platforms="zos_390-64.*" />
+		<exec command="tso delete J9CORE.DMP" platforms="zos_390-31.*" />
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core</arg>
+			<arg>$DUMPFILE$</arg>
+			<input>!findmodules package java/io</input>
```

Comment:
Shouldn't this be `java.io` (`.` instead of `/`)?

Reply:
It looks like `/` is required instead of `.`. I just tested manually:
```
> !findmodules package java/io
java.base                      !j9module 0x00007F92E808FA68
Found 1 module
> !findmodules package java.io
Found 0 modules
```
it seems like internally it is stored with a `\`: the `findmodules` command is comparing directly to the internal packageName:
```
return J9UTF8Helper.stringValue(packagePtr.packageName()).equals(targetName);
```
If this is not the desired functionality, !`findmodules package` will need to be modified

Reply:
This will be kept in mind for improving the user experience, but is not for now an issue.

Reply:
Yes, minimizing the translation required by humans would be good.